src/openrct2/ride/Ride.cpp

```diff
@@ -1433,7 +1433,7 @@ static void RideBreakdownUpdate(Ride& ride)
     // a 0.8% chance, less the breakdown factor which accumulates as the game
     // continues.
     if ((ride.reliability == 0
-         || static_cast<int32_t>(ScenarioRand() & 0x2FFFFF) <= 1 + RIDE_INITIAL_RELIABILITY - ride.reliability)
+         || static_cast<int32_t>(ScenarioRand() & 0x2FFFFF) <= 1 + kRideInitialReliability - ride.reliability)
```

Comment:
It's weird that this is cast to a signed int, but compared to an unsigned int.

Reply:
Good catch, I did make it cast to `uint32_t`.